app/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@tanstack/react-query": "4.0.10",
+    "@types/jest": "27.5.2",
+    "@types/node": "16.11.46",
+    "@types/react": "18.0.15",
+    "@types/react-dom": "18.0.6",
+    "classnames": "2.3.1",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "react-loading-skeleton": "3.1.0",
+    "react-router-dom": "6.3.0",
+    "react-scripts": "5.0.1",
+    "sass": "1.54.0",
+    "typescript": "4.7.4",
+    "web-vitals": "2.1.4"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject",
+    "lint": "npm run lint:js && npm run lint:styles && npm run prettier",
+    "lint:js": "eslint src --ext .js,.jsx,.ts,.tsx --fix",
+    "lint:styles": "stylelint '**/*.scss' --fix",
+    "prettier": "prettier --write 'src/**/*.{js,jsx,ts,tsx}'"
+  },
+  "browserslist": {
+    "production": [
+      ">0.2%",
+      "not dead",
+      "not op_mini all"
+    ],
+    "development": [
+      "last 1 chrome version",
+      "last 1 firefox version",
+      "last 1 safari version"
+    ]
+  },
+  "lint-staged": {
+    "src/**/*.{js,jsx,ts,tsx,json,css,scss,md}": [
+      "prettier --write"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-push": "lint-staged"
```

Comment:
Personally I would add for the pre-push: lint-staged && npm run test, in that way we can verify the unit test before proceeding to push every new change